### PR TITLE
SEO-187032-File-Formats-H1-Tag-Missing

### PR DESCRIPTION
**Pull request by EsterAtieno**
Hi @Yvone-Atieno ,
Have made changes,Kindly review.

Title | Description
-- | --
Task Category | Missing H1 Tag
Content Task Link/Mail Screenshot | NA
Hotfix | https://github.com/syncfusion-content/fileformat-docs/pull/2160
Ticket/Task link | https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/187032/
Work link | https://syncfusion.sharepoint.com/:x:/r/sites/SEOWebsiteAuditing/_layouts/15/Doc.aspx?sourcedoc=%7B2898E97F-22C5-48F0-B677-88AB8168596C%7D&file=Ester.xlsx&action=default&mobileredirect=true

Changes Made | Reason for change | Reason for not identifying earlier
-- | -- | --
Redirect link | To improve accessibility |  

Regards,
Ester

### CODE REVIEW

File: File-Formats/PDF/Working-with-PDF-Templates.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ platform: file-formats
 control: PDF
 documentation: UG
 ---
-# Working with PDF Templates 
+# Working with PDF Templates in .NET PDF
```
Comment (Yvone-Atieno) [inline File-Formats/PDF/Working-with-PDF-Templates.md:8]: Don't add PDF

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: File-Formats/PDF/Working-with-PDF-Templates.md (hunk 1 above)
Comment (EsterAtieno) [reply File-Formats/PDF/Working-with-PDF-Templates.md]: resolved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Pull request closed without merging**